### PR TITLE
Add UUID function

### PR DESCRIPTION
**Pull request by JuanMGN**
Add uuid support function according specification, https://www.w3.org/TR/sparql11-query/#func-uuid

### CODE REVIEW

**Comment by pepegar** [comment]
Seems that the files are not properly formatted.   We use scalafmt, scalafix, and scalastyle in CI to test that the style is correct.

You can fix this by running `sbt lint` directly on the repository :)

File: modules/engine/src/main/scala/com/gsk/kg/engine/QueryExtractor.scala [hunk 1]
```diff
@@ -253,6 +253,7 @@ object QueryExtractor {
       case ExpressionF.BOOL(s)             => s
       case ASC(e)                          => s"(asc $e)"
       case DESC(e)                         => s"(desc $e)"
+      case UUID()                          => "UUID()"
```
Comment (JuanMGN) [inline modules/engine/src/main/scala/com/gsk/kg/engine/QueryExtractor.scala:256]: I'm not sure if this change is neccessary, may I have to write (UUID) ??

Comment (pepegar) [reply modules/engine/src/main/scala/com/gsk/kg/engine/QueryExtractor.scala:256]: Yep, it should be `(uuid)`

**Review by pepegar — APPROVED** [review]
Apart of the QueryExtractor comment, looks good to me!

**Pull request merged**